scripts/provisioning/s3_setup

```diff
@@ -267,7 +220,14 @@ create_background_delete_account()
 if [ $s3postinstall == true ]
 then
     echo "INFO: Start haproxy, slapd, rabbitmq-server and rsyslog services.."
-    systemctl restart haproxy slapd rabbitmq-server rsyslog
+    sh /opt/seagate/cortx/s3/bin/s3prov_start_services.sh haproxy slapd rabbitmq-server rsyslog
+    if [ $? -eq 0 ]
+    then
+      echo "Services are running successfully."
+    else
+      echo "Failed to start services."
```

Comment:
I am not sure, but can we give info on which service failed to start !!!

Reply:
Yes, /opt/seagate/cortx/s3/bin/s3prov_start_services.sh already prints this information internally.